.codeclimate.yml

```diff
@@ -12,7 +12,7 @@ checks:
   file-lines:
     enabled: true
     config:
-      threshold: 250
+      threshold: 400
```

Comment:
Why did you increase the threshold? Put it back to 250.

Reply:
> Why did you increase the threshold? Put it back to 250.

i added the threshold because my component had more than 250 line